### DISCUSSION TITLE
Station for live-streaming only

### DISCUSSION THREAD

**Discussion by renggap** [AzuraCast/AzuraCast#5851] [Ideas]
Hello, 

Requesting a new feature: a station that broadcasts only live streaming from DJs. No AutoDJ involved since it gives us problems when the live source is disconnected, AutoDJ takes over, which gives bad impression to listeners. This feature can be found on other panel, hope to see it on AzuraCast too. 

Best regards, 
Rengga

**Comment by renatopinto89** [comment]
Just disable Auto-DJ on the Stations Profile page.
Important to note that if you´re using Shoutcast as a broadcasting backend, you´ll have to set the port to something XXX5 (example 8105) because Shoutcast v1 broadcasters, like VirtualDJ will use the legacy port+1 method.

**Reply by renggap** [reply]
Hi @renatopinto89, 

Thank you for your reply. I tested your solution and managed to get what I wanted to achieve. Disabled the AutoDJ feature on station profile, and changed port to 8xx5 (as same as DJ's port). Live streaming goes smoothly.

But when I switched back to AutoDJ mode (with the current port), AutoDJ won't start. The AutoDJ will work when I change the port back to initial (port 8xx0). My usage I wanted, a streamer can switch off AutoDJ completely when doing live, without changing the allocated port.

Is there any convenient way to achieve that?

**Reply by ChrisDuffley** [reply]
Not that I know of, unless you want to broadcast to Ice/Shoutcast directly? That may be your best friend here...

**Comment by alamunib** [comment]
I has tried to disable auto dj. But live source cant connected. Please check this bug. If auto dj off, live cant connected. If autodj on, live can connected. But if nothing source from live, auto dj return play. Can you give new feature autodj off live can connected. Thank you
